GameTests.java

```diff
@@ -1,14 +1,76 @@
 import org.junit.Test;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
 
 import static org.hamcrest.core.Is.is;
-import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
 
 public class GameTests {
     @Test
-    public void playerMoveIsRegistered() {
-        Game game = new Game(System.in);
+    public void playerMoveIsRegistered() throws IOException {
```

Comment:
I was always told to never to explicitly declare  throws Exception in a method signature.  I'm trying to find you some reasoning on it as I can't remember perfectly..

You also only have it on the first two tests and not the others in GameTest